sentry-android-core/src/main/java/io/sentry/android/core/ViewHierarchyEventProcessor.java

```diff
@@ -89,7 +93,9 @@ public static byte[] snapshotViewHierarchyAsData(
 
   @Nullable
   public static ViewHierarchy snapshotViewHierarchy(
-      @Nullable Activity activity, @NotNull ILogger logger) {
+      final @Nullable Activity activity,
+      final @NotNull ILogger logger,
+      final @NotNull List<ViewHierarchyExporter> exporters) {
```

Comment:
`m`: again this is a breaking change for RN - probably we should have overloads here rather? https://github.com/getsentry/sentry-react-native/blob/2d064085c1bb8e07ee516ecc6711232f42fbc5cc/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java#L411